src/audio/volume/volume.c

```diff
@@ -830,6 +832,9 @@ static int volume_prepare(struct comp_dev *dev)
 	cd->ramp_finished = true;
 	cd->vol_ramp_frames = dev->frames / (dev->period / VOL_RAMP_UPDATE_US);
 	cd->channels = sinkb->stream.channels;
+	cd->sample_rate = sinkb->stream.rate;
+	comp_info(dev, "volume_prepare(): channels = %d, vol_ramp_frames = %d",
```

Comment:
This trace need to be removed, I used it for debugging for correct stream parameters.

Reply:
Done.